README.md

```diff
@@ -783,73 +787,68 @@ Here are a few ways you might hear of or see in documents:
 - 🄳 A company is in
   [**liquidation overhang**](https://equityzen.com/blog/startup-valuations-and-liquidation-preference-overhang/)
-  when the value of a company doesn’t reach the number of dollars investors put into it.
+  when the value of the company doesn’t reach the dollar amount investors put into it.
   Because of liquidation preference, those holding preferred stock (investors) will have to
   be paid before those holding common stock (employees).
   If investors have put millions of dollars into your company and it’s sold, your equity as
   an employee
   [won’t be worth anything](https://avc.com/2010/10/employee-equity-the-liquidation-overhang/)
   if the company is in liquidation overhang and the sale doesn’t exceed that amount.
-- ☝️ Preferences are
-  [notoriously complex](https://venturebeat.com/2010/08/16/beware-the-trappings-of-liquidation-preference/).
-  We won’t cover all the details here, but generally, it’s worth understanding that
-  investors and entrepreneurs negotiate a lot of details, including:
-
+- ☝️ The complexities of the liquidation preference are
+  [infamous](https://venturebeat.com/2010/08/16/beware-the-trappings-of-liquidation-preference/).
+  It’s worth understanding that investors and entrepreneurs negotiate a lot of the details
+  around preferences, including:
   - The **multiple**, a number designating how many times the investor must be paid back before
-    common shareholders receive proceeds (often the multiple is 1X, but it can be 2X or
-    higher).
+    common shareholders receive proceeds.
+    (Often the multiple is 1X, but it can be 2X or higher.)
   - Whether preferred stock is
     [**participating**](https://en.wikipedia.org/wiki/Participating_preferred_stock), meaning
     investors get their money back and also participate in proceeds from common stock.
   - Whether there is a **cap**, which limits the payout if it is participating.
-
 - [This primer](https://medium.com/@CharlesYu/the-ultimate-guide-to-liquidation-preferences-478dda9f9332)
-  by Charles Yu gives a concise overview.
+  by Charles Yu gives a concise overview of preferred stock and the liquidation preference.
 - 🚧 What’s are good resources to mention that describe conversion of preferred to common
   stock?
 
-🔹 **What employees should know about preferences:**
-All this is complicated, but for the purposes of an employee who holds common stock, the
-most important thing to understand is that if a company does very well long term, these
-differences likely won’t matter as every stockholder has valuable stock they can
-eventually sell;
-but if a company fails or [exits](#sales-and-liquidity) for less than investors had hoped,
+🔹 For the **purposes of an employee who holds common stock**, the most important thing to
+understand about preferences is that they’re not likely to matter if a company does well
+in the long term.
+In that case, every stockholder will have the option to sell shares of some value.
```

Comment:
Revert. It now sounds like everyone has options (in the technical meaning) to sell shares, which is confusing. It was phrased the previous way to state: If you are a shareholder you likely can sell.

Reply:
Fixed.